src/bptl/work_units/xential/models.py

```diff
@@ -0,0 +1,24 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class XentialTicket(models.Model):
+    """
+    Store the relation between an executed task and a Xential ticket.
+    """
+
+    bptl_ticket_uuid = models.UUIDField(
+        _("BPTL ticket UUID"),
+        help_text=_("BPTL specific Xential ticket UUID"),
+    )
+    ticket_uuid = models.UUIDField(
+        _("Xential ticket UUID"), help_text=_("Xential ticket UUID")
+    )
+    task = models.ForeignKey("tasks.BaseTask", on_delete=models.PROTECT)
+
+    class Meta:
+        verbose_name = _("Xential Ticket")
+        verbose_name_plural = _("Xential Tickets")
+
+    def __str__(self):
+        return self.bptl_ticket_uuid
```

Comment:
Will probably raise an exception since it doesn't return a string, but a `uuid.UUID` instance.